bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+# kong
+
+[Kong](https://konghq.com/kong/) is a scalable, open source API layer (aka API gateway or API middleware) that runs in front of any RESTful API. Extra functionalities beyond the core platform are extended through plugins. Kong is built on top of reliable technologies like NGINX and provides an easy-to-use RESTful API to operate and configure the system.
+
+## TL;DR;
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/kong
```

Comment:
Please avoid including the `$`. This way is easier to copy/paste commands from the **README.md**.
Please apply this suggestion to the whole **README.md**.

Reply:
Done